ci-operator/step-registry/telco5g/cnf/rt-tests/telco5g-cnf-rt-tests-commands.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+function RTenable() {
+role=$1
+echo -e "enable RT on cluster \n"
+oc get nodes -o wide
+oc apply -f - <<_EOF
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  annotations:
+    kubeletconfig.experimental: |
+      {"podPidsLimit": 16384}
+  name: performance
+spec:
+  cpu:
+    isolated: 5-111
+    reserved: 0-4
+  machineConfigPoolSelector:
+    pools.operator.machineconfiguration.openshift.io/$role: ""
+  nodeSelector:
+    node-role.kubernetes.io/$role: ""
+  numa:
+    topologyPolicy: restricted
+  realTimeKernel:
+    enabled: true
+_EOF
+oc wait --for=condition=Updating --timeout=180s mcp $role
+sleep 10m
+oc wait --for=condition=Updated --timeout=30m mcp $role
+oc get nodes -o wide
+}
+
+
+function RTtest() {
+command=$1
+read -a command_arr <<< $command
+test=${command_arr[0]:2:-2}
+test=${test//_/-}
+
+echo -e "Run RT $test test with command $command \n"
+oc create -f - <<_EOF
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: $test
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: $test
+          image: quay.io/ocp-edge-qe/rt-tests@sha256:d0f1cba60f0ce896197139c8d5e04e963b4904b43076dc6018c5857491ff8855
+          imagePullPolicy: Always
+          command: $command
+          securityContext:
+            privileged: true
+            runAsNonRoot: false
+_EOF
+pod=$(oc get pod -o name |grep $test)
+oc wait $pod --for=jsonpath='{.status.phase}'=Succeeded --timeout=10m
+oc logs $pod
+}
+
+
+echo "**********************************************************************"
+oc get co
+echo "**********************************************************************"
+
+if [[ "$JOB_NAME" =~ "sno" ]]; then
```

Comment:
JOB_NAME will be the same in both case, bc ur calling the same ref 'telco5g-cnf-rt-tests'

Reply:
Ye thought about it too now, lets stay with env.

Reply:
Actually I mixed STEP_NAME with JOB_NAME.   But better to put parameter say: "SNO_CLUSTER: true"   default would be false. and only add env in sno workflow